### PR TITLE
 js_task_generate_table_DOM: Cypress tests added

### PR DESCRIPTION
**Pull request by iBrianWarner**

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (roman-mate) [inline cypress/fixtures/example.json:5]: is this needed?

File: cypress/integration/taskGenerateTableDom.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  });
+});
+
+describe('Genetate table app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should contain a first column as a Name', () => {
+    cy.tableInputs('Carolus Haverbeke', 'Emma de Milliano', 'Name');
```
Comment (roman-mate) [inline cypress/integration/taskGenerateTableDom.spec.js:24]: https://mate-academy.github.io/style-guides/javascript.html#naming--camelCase

`Function names are typically verbs or verb phrases.`

File: cypress/integration/taskGenerateTableDom.spec.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        expect(main).to.equal(child2);
+      });
+    });
+  });
+});
+
+describe('Genetate table app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
```
Comment (roman-mate) [inline cypress/integration/taskGenerateTableDom.spec.js:21]: https://docs.cypress.io/guides/references/best-practices.html#Setting-a-global-baseUrl

File: cypress/integration/taskGenerateTableDom.spec.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+  it('should contain a first column as a Name', () => {
```
Comment (roman-mate) [inline cypress/integration/taskGenerateTableDom.spec.js:23]: should we test by columns or rows?

**Review by roman-mate — CHANGES_REQUESTED** [review]

File: cypress/fixtures/example.json (hunk 1 above)
Comment (iBrianWarner) [reply cypress/fixtures/example.json:5]: nope, but it's always created then you run cypress. forgot to delete it.

File: cypress/integration/taskGenerateTableDom.spec.js (hunk 2 above)
Comment (iBrianWarner) [reply cypress/integration/taskGenerateTableDom.spec.js:24]: done

File: cypress/integration/taskGenerateTableDom.spec.js (hunk 4 above)
Comment (iBrianWarner) [reply cypress/integration/taskGenerateTableDom.spec.js:23]: some students create tables in desc order, so I decided to test columns

**Review by roman-mate — APPROVED** [review]

**Pull request merged**